nodes/nomos-node/node/src/api/testing/handlers.rs

```diff
@@ -66,12 +67,14 @@ where
     >(handle, session_id))
 }
 
+#[serde_as]
```

Comment:
Maybe: `use serde_with::{serde_as, Same};`

Reply:
Just change it directly to `Vec<(BlobId, SessionNumber)>` as it's only used for the post request.